query-engine/connectors/mongodb-query-connector/Cargo.toml

```diff
@@ -22,6 +22,7 @@ tracing-futures = "0.2"
 uuid.workspace = true
 indexmap = "1.7"
 query-engine-metrics = {path = "../../metrics"}
+cuid = { git = "https://github.com/prisma/cuid-rust", branch = "jkomyno/wasm32" }
```

Comment:
Any reason the branch name is under your name? Do you own it and will you be the sole maintainer of the branch, forever?

Reply:
Historical reasons: even the repo was `jkomyno/cuid-rust` before, because another `prisma/cuid-rust` fork existed.
I have already updated the reference repo, as you see, but I forgot about the branch name, sorry about that. Will update it to `wasm32`.

Reply:
Updated.